website/source/docs/agent/options.html.md

```diff
@@ -821,15 +821,14 @@ default will automatically work with some tooling.
 
     The following sub-keys are available:
 
-    * <a name="bootstrap"></a><a href="#config_entries_bootstrap">`bootstrap`</a>
-        This object allows configuring centralized config entries to be bootstrapped
-        by the leader. These entries will be reloaded during an agent config reload.
+    * <a name="config_entries_bootstrap"></a><a href="#config_entries_bootstrap">`bootstrap`</a>
+        This is a list of inlined config entries to insert into the state store when the Consul server
+        gains leadership. This option is only applicable to server nodes. Each bootstrap
+        entry will be created only if it does not exist. When reloading any newly entries
```

Comment:
```suggestion
        entry will be created only if it does not exist. When reloading, any new entries
```

Reply:
Not sure what happened but I have committed a fix for that typo twice now.